packages/flab/stringify.js

```diff
@@ -0,0 +1,9 @@
+console.log(
+	'module.exports = '+
+	require("fs").readFileSync("/dev/stdin", "utf-8")
```

Comment:
Ignore the `mockery` stuff.  The `memory-stream` package allows you to pass a stream object to a library and then call `.toString()` on what was written to the stream.